css/style.css

```diff
@@ -1,75 +1,101 @@
-.barchart {
-	max-width: 200px;
-	max-height: 350px;
+.server-info-wrapper {
+	max-width: 100vw;
 }
 
-.clipboardButton {
-	display: inline-block;
-	position: relative;
-	top: 2px;
-	margin-left: 5px;
+.table-wrapper {
+	overflow-x: auto;
+}
+
+.server-infos-table {
+	margin-bottom: 25px;
+}
+
+.server-infos-table td:first-child {
+	width: 120px;
+}
+
+.monitoring-wrapper {
+	display: -webkit-flex;
+	display: flex;
 }
 
-#ocsEndPoint {
-	margin-bottom: 10px;
+.monitoring-wrapper > input {
+	flex: 1 0 auto;
 }
 
-#monitoring-endpoint-url {
-	width: 80%;
-	max-width: 415px;
+.monitoring-wrapper + .settings-hint {
+	margin-top: 0;
 }
 
 .rambox {
-	padding: 0px;
-	margin: 0px;
- 	height: 10px;
- 	width: 10px;
- 	background-color: #555;
- 	border-radius: 20%;
+	height: 10px;
+	width: 10px;
+	background-color: #555;
 }
 
 .swapbox {
-  height: 10px;
-  width: 10px;
-  background-color: #222;
-  border-radius: 20%;
+	height: 10px;
+	width: 10px;
+	background-color: #222;
 }
 
 .info {
 	font-weight: bold;
-	text-color: #777777;
+	color: #777;
 }
 
 .smallinfo {
-	color: #888888;
+	color: #888;
 	font-size: 11px;
 }
 
 .infobox {
-	width: 400px;
-	border: 1px solid #DDDDDD;
-	margin: 10px 0px 20px 0px;
+	display: flex;
+	flex-wrap: nowrap;
+	align-items: center;
+
+	border: 1px solid #ddd;
 	padding: 15px;
-	color: #555555;
-	background-color: #F9F9F9;
+	margin: 15px 0;
+	color: #555;
+	background-color: #f9f9f9;
 	border-radius: 6px;
-	box-shadow: 1px 1px 9px 1px #EEEEEE;
+	box-shadow: 1px 1px 9px 1px #eee;
 }
 
-.infobox h3 {
-	color: #555555;
-	font-size: 22px;
-	font-weight: bold;
-	padding: 5px 0px 10px 0px;
-	margin: 0px;
+.diskchart-container {
+	width: 100px;
+	border: 1px black;
+}
+
+@media only screen and (max-width: 768px){
```

Comment:
we're now using 1024px.
You can rename the file to `.scss` and use the scss variable `$breakpoint-mobile` https://github.com/nextcloud/server/blob/2f27f122e167f27f1b09c3a4c9b70754ab953e88/core/css/variables.scss#L100

Reply:
see my commit below